activerecord/lib/active_record/sanitization.rb

```diff
@@ -54,7 +54,7 @@ def sanitize_sql_for_assignment(assignments, default_table_name = table_name)
       # Accepts an array, or string of SQL conditions and sanitizes
       # them into a valid SQL fragment for an ORDER clause.
       #
-      #   sanitize_sql_for_order(["field(id, ?)", [1,3,2]])
+      #   sanitize_sql_for_order([Arel.sql("field(id, ?)"), [1,3,2]])
```

Comment:
Is `Arel.sql` really necessary? None of the other examples in this file use it 🤔

Reply:
It is necessary. Since [317465](https://github.com/rails/rails/commit/317465a4a8f8c0ebaf9534921350c4540584c03c#diff-f483e172b4498ac82ab471a1b4eb97e025d0c68e63db5ff471993cff9be587ee), `order(["field(id, ?)", [1,3,2]])` will throw an `ActiveRecord::UnknownAttributeReference` error. eg:

![image](https://user-images.githubusercontent.com/33079237/121642366-00747e00-cac3-11eb-91b2-3f5632f4a066.png)

Reply:
@OuYangJinTing Is the same true of the other methods and examples?